safe/impact_functions/volcanic/volcano_building_impact.py

```diff
@@ -279,15 +279,17 @@ def run(self, layers):
             # Count affected buildings by usage type if available
             usage = None
             for building_type_attribute in building_type_attributes:
-                if building_type_attribute in attribute_names_lower and (
-                                usage is None or usage == 0):
+                if (
```

Comment:
So this is closest to our pull left policy. Also it has one bit of logic per line, which is close to the one item per line policy as requested above. 
Do you think we should prefer a different convention for logical statements?